src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -388,12 +389,30 @@ public void addStudent(Student student) {
      */
     public void removeStudent(Student student) {
         requireNonNull(student);
+        System.out.println("removing " + student.toString());
```

Comment:
remember to remove print statements hehe!

Reply:
Oops haha updated!! thanks much